src/MiniEstablishments/Market/marketEventData.ts

```diff
@@ -0,0 +1,201 @@
+        const merchantStance = ['not budging on the price', `looking like ${npc1.heshe} is losing this deal`, 'telling the customer to kindly shove it', 'offering a discount for a favor', 'holding firm on the price', 'actually managing to haggle for an even higher price', 'looking rather haggard']
+        return `${profile(npc1, lib.articles.output(npc1.descriptor).toUpperFirst())} is waving ${npc1.hisher} arms around in the air and curtly saying something to ${profile(npc2, lib.articles.output(npc2.descriptor))} who is holding a small bag of coins. The two appear to be haggling over the cost of ${lib.articles.output(lib.random(hagglingItem))}, and the merchant is ${lib.random(merchantStance)}.`
+      }
+    },
+    chase: {
+      function (town) {
+        const npc1 = createNPC(town, {
+          profession: 'merchant'
+        })
+        return `${profile(npc1, lib.articles.output(npc1.descriptor).toUpperFirst())} is waving ${npc1.hisher} fist in the air while running after a scrawny looking child. The merchant is yelling for someone to stop that theif.`
+      }
+    },
+    animalEscape: {
+      exclusions (town) {
+        return !town.bans.includes('animals')
+      },
+      function () {
+        const animal = ['bull', 'ox', 'lion', 'wolf', 'large boar', 'hippo', 'tiger', 'elephant', 'rhino', 'leapard', 'jaguar', 'hyena', 'grizzly bear', 'crocodile', 'alligator', 'killer bunny', 'giant scorpion', 'cougar', 'gorilla']
+        const chase = ['single child', 'group of armed thugs', 'few town guards', 'angry merchant', 'distraught merchant', 'party of adventurers', 'group of mercenaries', 'few shabby looking kids']
+        return `People around start to yell and shout as they realize ${lib.articles.output(lib.random(animal))} has escaped from its cage. The animal is going wild through the market as ${lib.articles.output(lib.random(chase))} chases after it.`
+      }
+    },
+    brawl: {
+      function (town) {
+        const npc1 = createNPC(town)
+        const npc2 = createNPC(town)
+        return `A crowd has gathered near the center of the market to watch a brawl that has broken out. ${profile(npc1, lib.articles.output(npc1.descriptor).toUpperFirst())} is in the center of the crowd currently holding ${profile(npc2, lib.articles.output(npc2.descriptor))} in a headlock and shouting something that can't be made out.`
+      }
+    },
+    music: {
+      function (town) {
+        const npc = createNPC(town, {
+          hasClass: true,
+          dndClass: 'bard'
+        })
+        const instrument = ['lute', 'pair of drumes', 'harp', 'flute', 'pan flute', 'sitar', 'fiddle', 'citern', 'vielle', 'clavichord', 'harpsichord']
+        const playing = ['rather well', 'rather poorly', 'very loudly', 'quite softly', 'and it sounds pretty good', 'and it sounds bad', 'and it sounds alright']
```

Comment:
You could use wageVariation as a measure of the bard's skill- see `createLifestyleStandards.ts` for an example of it being used. 
```
const sounding = [
  [25, 'and it sounds absolutely amazing'],
  [20, 'and it sounds great'],
  [0, 'quite loudly'],
  [-25, 'and it sounds like two cats having a fight']
] as ThresholdTable

  let note = sounding.find(desc => {
    return desc[0] >= wageVariation(town, npc)
  })
```